usb-gadget-service/scripts/iiod_context.sh

```diff
@@ -139,4 +139,14 @@ replace_or_add hw_serial "${SERIAL}"
 replace_or_add unique_id "${UNIQUE_ID}"
 replace_or_add dtoverlay "${OVERLAY}"
 
+EXTRA_EEPROM_FILE="/sys/devices/platform/soc/fe804000.i2c/i2c-1/1-0051/eeprom"
+if [ -f "$EXTRA_EEPROM_FILE" ]; then
+	while read -r LINE; do
+		IFS='=' read -r KEY VALUE <<< "$LINE"
```

Comment:
since the script references `#!/bin/sh`, the <<< redirection operator doesn't quite work. ("Syntax error: redirection unexpected")

This:
```
IFS='=' read -r KEY VALUE <<EOF
$LINE
EOF
```
Did the trick in my case.